package.json

```diff
@@ -25,6 +25,7 @@
 		"grunt-contrib-uglify": "^0.9.2",
 		"grunt-contrib-copy": "~0.4.1",
 		"grunt-contrib-clean": "~0.5.0",
+    "jit-grunt": "^0.9.1",
```

Comment:
@pra85 Please fix indentation and it's ready to be merged :)
Thanks for the patch!